app/build.gradle

```diff
@@ -5,7 +5,7 @@ android {
     defaultConfig {
         applicationId "com.macbitsgoa.events"
         manifestPlaceholders = [deepLinkDomain: "example.com"]
-        minSdkVersion 21
+        minSdkVersion 24
```

Comment:
why 24?

Reply:
This change is specific to my Android Studio, since for "minSdkVersion 21" it was giving a "dexing error" which I was unable to resolve.

Reply:
revert to 21 before final commit. (if possible use canary till next android studio release)